### ISSUE TITLE
Code changes to make Mozilla happy

### ISSUE THREAD

**Issue by twstokes** [twstokes/herpderp#13]
```
Hello,


Your add-on Herp Derp for YouTube™ has been reviewed and one or more of of its versions have been disabled due to critical issues discovered.

Details:
This version did not pass the review because of the following issues:

1) Recursive &#39;setTimeout&#39; function or &#39;setInterval&#39; with short intervals can impact browser performance. Please consider adjusting the interval and/or use &#39;clearInterval&#39; as soon as necessary.
eg:
herp.js#L41

Please fix them and submit again. Thank you.

Please note the following for the next update:
1)  Please avoid making global variables by declaring them with var/let/const
eg:
for (x=0; x&lt;=randomLength; x++)

Change to (for example):
for (let x=0; x&lt;=randomLength; x++)

Version(s) affected and disabled:
1.6.4

Reviewer:
erosman
```

**Comment by Shirkit** [comment]
Can you release a version for manual installation?

**Comment by jgillich** [comment]
+1, I need this addon.

**Comment by twstokes** [comment]
I've added a `builds` directory. You should be able to drag and drop it into Firefox. It's still a bit glitchy due to #12. I hope I can find time to work on it.

**Comment by twstokes** [comment]
These issues should be resolved with 1.6.5.
